independent-publisher-connectors/ScrapingBee/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1407 @@
+                                "title": "OG"
+                              }
+                            },
+                            "title": "Context"
+                          }
+                        }
+                      },
+                      "title": "Open Graph"
+                    },
+                    "dublincore": {
+                      "type": "array",
+                      "items": {
+                        "type": "object",
+                        "properties": {
+                          "elements": {
+                            "type": "array",
+                            "items": {
+                              "type": "object",
+                              "properties": {
+                                "name": {
+                                  "type": "string",
+                                  "description": "The name.",
+                                  "title": "Name"
+                                },
+                                "content": {
+                                  "type": "string",
+                                  "description": "The content.",
+                                  "title": "Content"
+                                },
+                                "URI": {
+                                  "type": "string",
+                                  "description": "The URI.",
+                                  "title": "URI"
+                                }
+                              }
+                            },
+                            "title": "Elements"
+                          },
+                          "terms": {
+                            "type": "array",
+                            "items": {
+                              "type": "string"
+                            },
+                            "description": "The .",
+                            "title": "Terms"
+                          },
+                          "@context": {
+                            "type": "object",
+                            "properties": {},
+                            "description": "The context.",
+                            "title": "Context"
+                          }
+                        }
+                      },
+                      "title": "Dublincore"
+                    }
+                  },
+                  "title": "Metadata"
+                }
+              }
+            }
+          }
+        },
+        "description": "Fetches the URL requested to scrap and will render JavaScript if requested.",
+        "x-ms-visibility": "important"
+      }
+    },
+    "/v1/usage": {
+      "get": {
+        "summary": "Get usage",
+        "tags": [
+          "HTML API"
+        ],
+        "operationId": "UsageInformation",
+        "deprecated": false,
+        "parameters": [],
+        "responses": {
+          "200": {
+            "description": "The .",

```

Comment:
@troystaylor can you please double check the description here! it looks like it is incomplete.